From 49c9372b6d767fd537e6016ca33785c14f7301cf Mon Sep 17 00:00:00 2001
From: Chunrong Guo <chunrong.guo@nxp.com>
Date: Mon, 7 May 2018 16:50:47 +0800
Subject: [PATCH] Makefile: fix build error

fix the below error:
/bin/sh: whoami: command not found

Upstream-Status: Inappropriate [embedded-specific]

Signed-off-by: Chunrong Guo <chunrong.guo@nxp.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/import/Makefile b/src/import/Makefile
index d9b472a..e87bbd8 100644
--- a/src/import/Makefile
+++ b/src/import/Makefile
@@ -40,7 +40,7 @@ goenv:
 		wget -c https://redirector.gvt1.com/edgedl/go/$(GOFILE); \
 		tar -C $(HOME) -xzf $(GOFILE); \
 	fi
-	usr=`whoami`; \
+
 
 clean:
 	for dir in $(TARGETS);\
-- 
1.8.3.1

